crates/typst-library/translations/eu.txt

```diff
@@ -0,0 +1,8 @@
+figure = Irudia
+table = Taula
+equation = Ekuazioa
+bibliography = Bibliografia
+heading = Atala
+outline = Aurkibidea
+raw = Kodea
+page = Orria
```

Comment:
In the [Spanish translation](https://github.com/typst/typst/blob/main/crates/typst-library/translations/es.txt) "page" is written in lowercase. Should that be the case here as well?

Reply:
Could be. It depends if it is at the beginning of a sentence. The same for section, raw, and everything else.

Besides, in Spanish raw is also poorly translated. I plan to fix it.

Reply:
> It depends if it is at the beginning of a sentence. The same for section, raw, and everything else.

It's not possible to distinguish for now. You should assume that it's used within the sentence. At the start of the sentence would be less common and needs to be handled manually.

Reply:
It's OK then. I think the PR is ready for supporting basque. I would investigate on the hyphen part later. Thanks for the comments.